app/models/page.rb

```diff
@@ -89,6 +89,14 @@ class Page < ApplicationRecord
 
   serialize :metadata, Hash
 
+  ACCEPTED_FILE_TYPES = [
+    'image/jpeg',
+    'image/png',
+    'image/gif',
+    'image/bmp',
+    'image/tiff'
+  ].freeze
```

Comment:
Please review if I have covered all accepted files for page images here

Reply:
[Documentation says](https://content.fromthepage.com/project-owner-documentation/image-upload-file-guidelines/):  Acceptable file formats: PDF, PNG, GIF, JPG, TIF

We're not supporting PDF for an individual page upload, but we should support the others.  Are we transforming them?  I think we do transform some if they are uploaded in a zip file for an initial upload.

Reply:
Currently our filepicker component only accepts image files

Reply:
In any case, #3890 can add more supported files here